source/common/runtime/runtime_features.cc

```diff
@@ -49,13 +51,24 @@ constexpr const char* disabled_runtime_features[] = {
     "envoy.reloadable_features.http2_protocol_options.stream_error_on_invalid_http_messaging",
 };
 
+// Finally this is a section for runtime features which used to exist and have
+// been removed in the last few years. It exists to allow best-effort attempts
+// to warn folks to clean up use of removed runtime guards.
```

Comment:
Oh, hah. This is not what I thought you were talking about, though still very nice. :) I thought we were talking about spitting out a warning when someone overrides runtime for a feature that has been disabled by default. Do you want to throw that in here as well? I feel like we should remind people with a warning that they are doing something they shouldn't be? Or do we already do that?

Reply:
Yeah if you want to remove this part of the change that's fine with me. Not sure if it's worth carrying the code around, but I don't feel strongly about it either way.